src/main/java/uk/gov/hmcts/reform/sscs/service/citizenrequest/CitizenRequestService.java

```diff
@@ -125,8 +127,17 @@ private String getSubscriptionEmail(Subscription subscription) {
         return subscription != null ? subscription.getEmail() : null;
     }
 
-    private boolean submitHearingRecordingRequest(SscsCaseData sscsCaseData, Long ccdCaseId, List<String> hearingIds, String idamEmail) {
+    private boolean submitHearingRecordingRequest(long ccdCaseId, List<String> hearingIds, String idamEmail) {
```

Comment:
Does long need to be capitalised or is the behaviour the same with lower case?

Reply:
changed it back to Long